src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -281,10 +308,14 @@ class MockClientStats {
     auto time = std::chrono::system_clock::now();
     request_timestamps.push_back(time);
 
+    // Group all values across all inputs together into a single vector, and
+    // then record it
+    std::vector<TestRecordedInput> tmp_inputs;
     for (const auto& input : inputs) {
-      recorded_inputs.push_back(
-          static_cast<const MockInferInput*>(input)->recorded_inputs_);
+      auto x = static_cast<const MockInferInput*>(input)->recorded_inputs_;
```

Comment:
I know this is temporary variable but variable names like `x` tell me nothing. `tmp_inputs` atleast tells me that it is a temporary value. `x` is pretty throw away. 
This looks like it would be a single `recorded_input`. Can we change it to that?

Reply:
renamed